src/types/type_param.rs

```diff
@@ -84,35 +102,110 @@ pub enum TypeArg {
         arg: CustomTypeArg,
     },
     /// Instance of [TypeParam::List] or [TypeParam::Tuple], defined by a
-    /// sequence of arguments.
+    /// sequence of elements.
     Sequence {
         #[allow(missing_docs)]
-        args: Vec<TypeArg>,
+        elems: Vec<TypeArg>,
```

Comment:
This was a driveby, see separate commit 405d362e1ba63de3009a6134a2141d65580bc054